prog/vm/github_runner.rb

```diff
@@ -227,7 +227,7 @@ def before_run
     case vm.sshable.cmd("systemctl show -p SubState --value #{SERVICE_NAME}").chomp
```

Comment:
fyi, it's "exited" not "exitted" (though it's a defensible extrapolation of the rules of english, were there such a thing). See commit message.

Reply:
I went ahead and ran the commit message through a spell check

Reply:
Thanks!